src/NServiceBus.Core/Routing/MessageEndpointMappingCollection.cs

```diff
@@ -175,6 +175,10 @@ internal void Apply(Publishers publishers, UnicastRoutingTable unicastRoutingTab
             {
                 m.Configure((type, endpointAddress) =>
                 {
+                    if (!conventions.IsMessageType(type))
```

Comment:
What do you mean? The MEMs are read from the config file. The thing calling the lambda does not check the conventions.